lib/spotify_sdk_web.dart

```diff
@@ -735,6 +745,16 @@ external set _onSpotifyWebPlaybackSDKReady(void Function()? f);
 @JS('onSpotifyWebPlaybackSDKReady')
 external void Function()? get _onSpotifyWebPlaybackSDKReady;
 
+/// Allows assigning the function getOAuthToken
```

Comment:
```suggestion
/// Allows getting the function getOAuthToken
```
Since this is for the getter shouldn't that be `getting`?